pkg/handlers/options/global.go

```diff
@@ -33,7 +35,7 @@ func (o *GlobalOptions) AddFlags(flags *pflag.FlagSet) {
 }
 
 func (o *GlobalOptions) DefaultsNamespace() string {
-	return o.defaultsNamespace
+	return cmp.Or(o.defaultsNamespace, corev1.NamespaceDefault)
```

Comment:
Defaulting here too because unit tests are now calling `NewGlobalOptions()` directly without calling  `AddFlags`.

Reply:
Instead of this, we can set the default in `NewGlobalOptions` and use that property in the default for `defaults-namespace` flag.

Reply:
Done in https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/562